include/_gtia.h

```diff
@@ -61,14 +75,47 @@ struct __gtia_write {
     unsigned char   colpf2; /* 0x18: color playfield 2 */
     unsigned char   colpf3; /* 0x19: color playfield 3 */
     unsigned char   colbk;  /* 0x1A: color background */
+
+/* See the "HUE_..." #defines in "atari.h" for the hue values to use with color registers, above */
+/* Bitwise OR (|) with 0x00 (darkest) through 0x0F (lightest) (only even values are unique) */
+
     unsigned char   prior;  /* 0x1B: priority selection */
-    unsigned char   vdelay; /* 0x1C: vertical delay */
+
+#define PRIOR_P03_PF03          0x01 /* Players 0-3, then Playfields 0-3, then background */
+#define PRIOR_P01_PF03_P23      0x02 /* Players 0-1, then Playfields 0-3, then Players 2-3, then background */
+#define PRIOR_PF03_P03          0x04 /* Playfields 0-3, then Players 0-3, then background */
+#define PRIOR_PF01_P03_PF23     0x08 /* Playfields 0-1, then Players 0-3, then Playfields 2-3, then background */
+
+#define PRIOR_5TH_PLAYER        0x10 /* Four missiles combine to be a 5th player (uses COLPF3) */
+#define PRIOR_OVERLAP_3RD_COLOR 0x20 /* Overlap of players 0 and 1, and of players 2 and 3, results in a third color
+                                        (else overlap is black). The resulting color is a logical OR of the two player colors. */
```

Comment:
Please follow the (very unusual) style for multi-line comments used in the cc65 code base.